src/test/java/org/springframework/data/jpa/repository/sample/UserRepository.java

```diff
@@ -290,6 +290,12 @@
 	// DATAJPA-496
 	List<User> findByAttributesIn(Set<String> attributes);
 
+	List<User> findByAttributesIgnoreCaseIn(Collection<String> attributes);
```

Comment:
Please move those methods to the bottom of the file and mark them with the JIRA ticket because of which they were introduced.